folium/folium.py

```diff
@@ -676,11 +663,9 @@ def _popup_render(self, popup=None, mk_name=None, count=None,
             Type of marker. Simple, Circle, etc.
         count: int, default None
             Count of marker
-        popup_on: boolean, default True
-            If False, no popup will be rendered
         """
-        if not popup_on:
-            return 'var no_pop = null;'
+        if not popup:
+            return ''
```

Comment:
In fact, in my tests, that behaviour happens anyways, even if `no_popup` is set. Try the same on the master branch and you'll see the hand popping up anyways, even if popup_on is set to False.